control/plugin/client/grpc.go

```diff
@@ -132,7 +135,13 @@ func NewCollectorGrpcClient(address string, timeout time.Duration, security GRPC
 
 // NewStreamCollectorGrpcClient returns a stream collector gRPC client
 func NewStreamCollectorGrpcClient(address string, timeout time.Duration, security GRPCSecurity) (PluginStreamCollectorClient, error) {
-	p, err := newPluginGrpcClient(address, timeout, security, plugin.StreamCollectorPluginType)
+	header := metadata.New(map[string]string{
+		"MaxCollectDuration": "5s",
```

Comment:
Thanks for the comment. The idea was to pass just the task-id. I've added these other parameters for testing purposes but we currently don't receive them anywhere. So I'll go ahead and remove these.